pkg/limitador/k8s_objects.go

```diff
@@ -219,8 +218,8 @@ func ValidatePDB(pdb *policyv1.PodDisruptionBudget) error {
 
 func Labels(limitador *limitadorv1alpha1.Limitador) map[string]string {
 	return map[string]string{
-		"app":                "limitador",
-		"limitador-resource": limitador.ObjectMeta.Name,
+		helpers.LabelKeyApp:               "limitador",
```

Comment:
we may not need this change.

Reply:
@eguzki are you fine with this being included. the value is only used once but multiple times in tests. it doesn't change any logic.